cmd/osbuild-worker/main.go

```diff
@@ -440,6 +440,10 @@ var run = func() {
 				Solver:               solver,
 				RepositoryMTLSConfig: repositoryMTLSConfig,
 			},
+			worker.JobTypeSearch: &SearchJobImpl{
```

Comment:
Nitpick: IMHO, it would make sense to include "package" in the job name since we may end up with more than just package search jobs in the future. The API endpoint also contains "package" in the path.

Reply:
Yeah, I thought about that but opted for the shorter string :) I'll take a poll, if people want `SearchPackageJob` instead click the button :)

Reply:
Which button should I click? :) I'm +1 for `SearchPackageJob`, `SearchPackageJobResult` etc

Reply:
Or `PackageSearchJob` :innocent: